docs/docs/end-users/Configuration.fsx

```diff
@@ -0,0 +1,166 @@
+(**
+---
+category: End-users
+categoryindex: 1
+index: 3
+---
+*)
+#r "nuget: Fantomas.Core, 5.0.0-alpha-*"
+
+open Fantomas.Core.FormatConfig
+open Fantomas.Core
+
+(**
+# Configuration
+Fantomas ships with a series of format options.
+These can be stored in an [.editorconfig](https://editorconfig.org/) file and will be picked up automatically by the 
+commandline. <br>
+Your IDE should respect your settings, but the implementation of that is editor specific. Setting the configuration via 
+UI might be available depending on the IDE but do note that these features might not be up to 
```

Comment:
UI might be available depending on the IDE. <strike>but do note that these features might not be up to...</strike>